tools/nimgrep.nim

```diff
@@ -62,10 +62,11 @@ Options:
        -s[:asc|desc]  - ascending (default: recent files go last) or descending
   --group, -g         group matches by file
   --newLine, -l       display every matching line starting from a new line
-  --limit[:N], -m[:N] limit max width of lines from files by N characters (80)
-  --fit               calculate --limit from terminal width for every line
-  --onlyAscii, -o     use only printable ASCII Latin characters 0x20-0x7E
-                      (substitutions: 0 -> @, 1-0x1F -> A-_, 0x7F-0xFF -> !)
+  --cols[:N]          limit max width of lines from files by N characters (off)
+  --cols:auto, -%     calculate columns from terminal width for every line
+  --onlyAscii, -@     use only printable ASCII Latin characters 0x20-0x7E
```

Comment:
optional nit: rename to `--ascii` ?

Reply:
I'm going to leave -@ and -% for brevity. I really use these ones lately. There are often cases when you need to work with nimgrep almost interactively: when previous search gave too huge results, you go to previous cmdline and add -@% option in 4 key presses.